### PR TITLE
Assignment3 commit

### PR DESCRIPTION
**Pull request by unnikrishnanks-1**
Method which would return an index of an alphabet in a string.
Eg : Find the index of x,a,o  in the given name, string name = “Xioami”

### CODE REVIEW

File: src/test/java/com/training/sample/tests/Assignment3Test.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.junit.jupiter.api.Test;
+
+public class Assignment3Test {
+    Assignment3 assignment3 = new Assignment3();
+
+    @Test
+    public void testGetIndex() {
+        String firstString = "X";
+        String secondString = "a";
+        String thirdString = "o";
```
Comment (PranavHanwante) [inline src/test/java/com/training/sample/tests/Assignment3Test.java]: direclty use in assertion

File: src/main/java/com/training/sample/pages/Assignment3.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.training.sample.pages;
+
+public class Assignment3 {
+    public int getIndex(String fullstring, String character) {
```
Comment (PranavHanwante) [inline src/main/java/com/training/sample/pages/Assignment3.java]: String character ?

File: src/test/java/com/training/sample/tests/Assignment3Test.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import org.junit.jupiter.api.Test;
+
+public class Assignment3Test {
+    Assignment3 assignment3 = new Assignment3();
+
+    @Test
+    public void testGetIndex() {
+        Assertions.assertEquals(0, assignment3.getIndex("Xioami", "X"));
+        Assertions.assertEquals(3, assignment3.getIndex("Xioami", "a"));
+        Assertions.assertEquals(2, assignment3.getIndex("Xioami", "o"));
```
Comment (PranavHanwante) [inline src/test/java/com/training/sample/tests/Assignment3Test.java]: Xiaomi more than 1 times, assign it in a variable

File: src/main/java/com/training/sample/pages/Assignment3.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package com.training.sample.pages;
+
+public class Assignment3 {
+    public int getIndex(String fullstring, String alphabet) {
```
Comment (PranavHanwante) [inline src/main/java/com/training/sample/pages/Assignment3.java]: fullstring =  camel casing

**Review by akhil7000 — APPROVED** [review]

File: src/main/java/com/training/sample/pages/Assignment3.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+package com.training.sample.pages;
+
+public class Assignment3 {
+    public int getIndex(String fullString, String alphabet) {
```
Comment (PranavHanwante) [inline src/main/java/com/training/sample/pages/Assignment3.java]: change name fullString

**Review by PranavHanwante — APPROVED** [review]

**Pull request merged**